README.md

```diff
@@ -8,7 +8,8 @@
 [![Follow on Twitter](https://badgen.net/twitter/follow/ahujaankit)](https://twitter.com/ahujaankit)
 ![License](https://img.shields.io/github/license/ankit/stylebot)
 
-Stylebot is a browser extension that lets you change the appearance of the web instantly. Available on [Firefox](https://addons.mozilla.org/en-US/firefox/addon/stylebot-web/) and [Chrome](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha).
+Stylebot is a browser extension that lets you change the appearance of the web instantly.
+Available on [Chrome](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/stylebot-web/) and [Edge](https://microsoftedge.microsoft.com/addons/detail/stylebot/mjolbpfednnbebfapicajpifliopnnai).
 
 - Easy to use: Pick and style elements using UI actions
 - Simple & Quick: Changes are saved instantly
```

Comment:
{"version":"2.0.104.12","settings":{"blur":0,"brightness":100,"contrast":100,"grayscale":0,"huerotate":0,"invert":0,"saturate":100,"sepia":0,"applyvideofilters":false,"backgroundcolor":"#000000","backgroundopacity":50,"blackbars":false,"blockads":false,"blockadsexceptforsubs":true,"blockautoplay":true,"blockhfrformats":false,"blockwebmformats":false,"cinemamode":false,"cinemamodewideplayer":false,"controlbar":{"active":true,"autohide":false,"centered":false,"position":"absolute"},"controls":["loop","stop","reverse-playlist","volume-booster","whitelist","not-interested","cards-end-screens","cinema-mode","size","pop-up-player","speed-minus","speed","speed-plus","video-filters","flip-horizontally","flip-vertically","screenshot","keyboard-shortcuts","custom-script","options"],"controlsvisible":false,"controlspeed":false,"controlspeedmousebutton":false,"controlvolume":true,"controlvolumemousebutton":false,"customcolors":{"--main-color":"#00adee","--main-background":"#111111","--second-background":"#181818","--hover-background":"#232323","--main-text":"#eff0f1","--dimmer-text":"#cccccc","--shadow":"#000000"},"customcssrules":"","customscript":"","customtheme":false,"darktheme":true,"date":1624501756028,"defaultvolume":false,"disableautoplay":true,"executescript":false,"expanddescription":false,"filter":"none","hidecardsendscreens":false,"hidechat":false,"hidecomments":false,"hiderelated":false,"ignoreplaylists":false,"ignorepopupplayer":false,"localecode":"en_US","localedir":"ltr","message":false,"miniplayer":false,"miniplayerposition":"_top-left","miniplayersize":"_400x225","newestcomments":false,"overridespeeds":false,"pauseforegroundtab":false,"pausevideos":false,"popuplayersize":"640x360","qualityembeds":"medium","qualityembedsfullscreen":"hd1080","qualityplaylists":"hd720","qualityplaylistsfullscreen":"hd1080","qualityvideos":"hd720","qualityvideosfullscreen":"hd1080","reload":false,"reversemousewheeldirection":false,"selectquality":false,"selectqualityfullscreenoff":true,"selectqualityfullscreenon":false,"speed":1,"speedvariation":0.05,"stopvideos":false,"theatermode":false,"theme":"default-dark","themevariant":"youtube-deep-dark-9anime.css","update":0,"volume":50,"volumemultiplier":10,"volumevariation":5,"whitelist":"","wideplayer":false,"wideplayerviewport":false}}